### ISSUE TITLE
Enhancement: Add option for IP Anonymization 

### ISSUE THREAD

**Issue by Verbat** [rainlab/googleanalytics-plugin#49]
In some countries it is legaly required to use ip anonymization when using Google Analytics.
For now i override the default markup with my own partial but it would be of great help if this setting could be toggled on via the backend.
https://developers.google.com/analytics/devguides/collection/analyticsjs/ip-anonymization

**Comment by jBOKA** [comment]
Can you post the alternative markup with the anonymization line wrapped in an twig `if`, I will then prepare the settings and create a pull request.

I also neeed this option.

**Comment by daftspunk** [comment]
Yes, please show us how you did this or submit a PR to have it implemented.

**Comment by Verbat** [comment]
I created a new partial in analytics/default.htm to override the default one.
My partial contains the following.

```
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

 ga('create', '{{ __SELF__.trackingId }}', '{{ __SELF__.domainName }}');
 ga('set', 'anonymizeIp', true);
 ga('send', 'pageview');

</script>

```

Basically, to use ip anonymization you only need to add the follwing:
 **ga('set', 'anonymizeIp', true);**

**Comment by jBOKA** [comment]
Here we go
PR https://github.com/rainlab/googleanalytics-plugin/pull/59

**Comment by jBOKA** [comment]
push: @daftspunk can you merge please

**Comment by jBOKA** [comment]
push

**Comment by LukeTowers** [comment]
Fixed by #59
